sdk/iot/Azure.Iot.Hub.Service/tests/ConfigurationsClientTests.cs

```diff
@@ -0,0 +1,150 @@
+                Assert.AreEqual(updatedConfig.Value.Priority, testPriority, "Priority should have been updated.");
+            }
+            finally
+            {
+                // Delete twin configuration
+                await Cleanup(client, twinConfiguration);
+            }
+        }
+
+        /// <summary>
+        /// Test get configurations
+        /// </summary>
+        [Test]
+        public async Task ConfigurationsClient_GetConfigurations()
+        {
+            const int configurationsCount = 5;
+            TwinConfiguration[] twinConfigurations = new TwinConfiguration[configurationsCount];
+            TwinConfiguration[] createdConfigurations = new TwinConfiguration[configurationsCount];
+            IReadOnlyList<TwinConfiguration> listConfigurations;
+            IotHubServiceClient client = GetClient();
+            try
+            {
+                for (int i = 0; i < configurationsCount; i++)
+                {
+                    twinConfigurations[i] = CreateTestConfig($"testconfigurations{i}{GetRandom()}");
+
+                    // Create Configurations
+                    Response<TwinConfiguration> createResponse =
+                        await client.Configurations.CreateOrUpdateConfigurationAsync(twinConfigurations[i]).ConfigureAwait(false);
+                    createdConfigurations[i] = createResponse.Value;
+                }
+
+                // List the configurations for the client
+                listConfigurations = (await client.Configurations.GetConfigurationsAsync().ConfigureAwait(false)).Value;
+
+                IEnumerable<string> twinConfigurationsIds = listConfigurations.ToList().Select(configuration => configuration.Id);
+
+                // Compare the response ids with created configurations
+                for (int i = 0; i < configurationsCount; i++)
+                {
+                    Assert.IsTrue(twinConfigurationsIds.Contains(twinConfigurations[i].Id));
+                }
+            }
+            finally
+            {
+                for (int i = 0; i < configurationsCount; i++)
+                    await Cleanup(client, twinConfigurations[i]);
+            }
+        }
+
+        private async Task Cleanup(IotHubServiceClient client, TwinConfiguration config)
+        {
+            // cleanup
+            try
+            {
+                if (config != null)
+                {
+                    await client.Configurations.DeleteConfigurationAsync(config, IfMatchPrecondition.UnconditionalIfMatch).ConfigureAwait(false);
+                }
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail($"Test clean up failed: {ex.Message}");
+            }
+        }
+
+        private TwinConfiguration CreateTestConfig(string testConfigurationId)
+        {
+            TwinConfiguration twinConfiguration =
```

Comment:
you can use var twinConfiguration  = as right hand side already tells the type.

Reply:
will do